app/client/components/Upload.tsx

```diff
@@ -189,7 +189,12 @@ export class Upload extends React.Component<UploadProps, UploadState> {
     }
 
     private handleTitle(e) {
-        this.setOk("videoTitle");
+        const title: string = e.target.value;
+        if (VIDEO_TITLE_RE.test(title)) {
+            this.setOk("videoTitle");
+        } else {
+            this.setErr("videoTitle", "error", "Video titles must contain letters, numbers, and spaces.");
```

Comment:
"must contain only letters, numbers, and spaces" might make it a little clearer.